app/src/main/java/com/example/ithaca_transit_android_v2/views/DrawRouteCard.kt

```diff
@@ -86,18 +125,62 @@ class DrawRouteCard(context: Context, attrs: AttributeSet?) : View(context, attr
 
     }
 
+    fun setDrawWalking(){
+
+        onlyWalking = true;
+
+        circleY2 = 150f
+
+
+    }
+
+
     override fun onDraw(canvas: Canvas) {
-        canvas.drawCircle(circleX1, circleY1, circleR, blueDotColor)
+        //reset
+        if(onlyWalking){
+            canvas.drawCircle(circleX1, circleY1, circleR, grayDotColor)
```

Comment:
if these are grouped purposefully, it might be helpful to add comments for each grouping or the most important group